pimd/pim_igmp.c

```diff
@@ -753,6 +753,39 @@ static void igmp_group_free(struct igmp_group *group)
 	XFREE(MTYPE_PIM_IGMP_GROUP, group);
 }
 
+static void igmp_group_count_incr(struct igmp_sock *igmp)
+{
+	struct pim_interface *pim_ifp = igmp->interface->info;
+
+	if (!pim_ifp)
+		return;
+
+	++pim_ifp->pim->igmp_group_count;
+	if (pim_ifp->pim->igmp_group_count
+	    == pim_ifp->pim->igmp_watermark_limit) {
+		zlog_warn(
+			"IGMP-WM-WARN: igmp group count reached watermark limit: %u(vrf: %s)",
```

Comment:
IGMP-WM-WARN is this really helpful?

Reply:
Question is still valid :)

Reply:
@ton31337 Sorry I didn't notice this. Earlier you had asked me to change from SCALE ALERT. 
My thought behind this was to grep these warnings using a identifier. I guess some vendor (mostly likely cisco) has it.
Now I have removed this.